lib/measures/tbd/resources/psi.rb

```diff
@@ -161,7 +160,7 @@ def complete?(s)
   end
 end
 
-def processTBDinputs(surfaces, edges, set, io_path, schema_path)
+def processTBDinputs(surfaces, edges, set, io_path = nil, schema_path = nil)
```

Comment:
@rd2 I propose making `io_path` and `schema_path` optional arguments with default of `nil`

Reply:
OK.